### PR TITLE
fix #38725 camwhores.tv

### LINKED ISSUE #38725: camwhores.tv
### Issue URL (Ads)
[NSFW][http://www.camwhores.tv/videos/4953586/innocent-enna-2019-08-13-12-39-36-212-001/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=http%3A%2F%2Fwww.camwhores.tv%2Fvideos%2F4953586%2Finnocent-enna-2019-08-13-12-39-36-212-001%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
Platform: | Win
AdGuard version: | 7.1.2817.0
Browser: | Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:68.0) Gecko/20100101 Firefox/68.0
AdGuard driver: | WFP
Stealth mode options: | Block Push API
Filters: | AdGuard Russian,<br>Filter unblocking search ads and self-promotion,<br>EasyList Dutch,<br>Liste FR,<br>ROList,<br>AdGuard Annoyances,<br>AdGuard French,<br>AdGuard Base,<br>ROLIST2,<br>road-block,<br>NoCoin Filter List,<br>Frellwit's Swedish Filter,<br>Dandelion Sprout's Nordic Filters,<br>AdGuard Tracking Protection,<br>AdGuard Social Media,<br>AdGuard Experimental,<br>AdGuard German,<br>AdGuard Japanese

### PR DESCRIPTION
**Pull request by zloyden**
https://github.com/AdguardTeam/AdguardFilters/issues/38725

### CODE REVIEW

File: EnglishFilter/sections/css_extended.txt [hunk 1]
```diff
@@ -675,6 +675,9 @@ theverge.com##.l-col__main > div.c-compact-river > div.c-compact-river__entry[-e
 toolslib.net##[class="col-lg-4 col-md-6 col-sm-6"][-ext-has='.custom > div.custom-label > span:contains(Advertising)']
 ! https://forum.adguard.com/index.php?threads/23429/
 bhg.com.au##.OUTBRAIN ul.ob-widget-items-container > li.ob-dynamic-rec-container[-ext-has='a[onmousedown*="https://paid.outbrain.com/network/redir?"]']
+! https://github.com/AdguardTeam/AdguardFilters/issues/38725
+!+ PLATFORM(ext_ff, ext_opera)
+camwhores.tv#$?#body > div[style*="position"][style*="box-shadow"]:has(> a[href^="https://t.hrtyj.com/"]) { position: absolute!important; left: -3000px!important; }
```
Comment (BlazDT) [inline EnglishFilter/sections/css_extended.txt]: Not shown in code for me.

Comment (Alex-302) [reply EnglishFilter/sections/css_extended.txt]: better `camwhores.tv##body > div[style^="position: fixed; top: 50%; left: 50%; transform: translate(-50%, -50%); width: 60%; height: 60%;"]`

Comment (Alex-302) [inline EnglishFilter/sections/css_extended.txt]: better `camwhores.tv##body > div[style^="position: fixed; top: 50%; left: 50%; transform: translate(-50%, -50%); width: 60%; height: 60%;"]`

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: EnglishFilter/sections/css_extended.txt (hunk 1 above)
Comment (zloyden) [reply EnglishFilter/sections/css_extended.txt]: If I hide that element, I can't press on button player. That's why I use css rule.

Comment (zloyden) [reply EnglishFilter/sections/css_extended.txt]: <details><summary>Screenshot:</summary>


</details><br/>

Comment (Alex-302) [reply EnglishFilter/sections/css_extended.txt]: ok

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**